v0/destinations/algolia/transform.js

```diff
@@ -22,10 +28,20 @@ const {
   clickPayloadValidator
 } = require("./util");
 
+const DestTfError = new ErrorBuilder()
```

Comment:
We shouldn't be reusing the ErrorBuilder instance. Perhaps a better way to achieve the same is:

```
const DefStatTags = {
      destType: DESTINATION,
      stage: TRANSFORMER_METRIC.TRANSFORMER_STAGE.TRANSFORM,
      scope: TRANSFORMER_METRIC.MEASUREMENT_TYPE.TRANSFORMATION.SCOPE,
    };
.
.
.
// some point in the code,
// throw an actual error object
// either overriding defaults or adding extra info
throw new ErrorBuilder()
  .setStatTags(DefStatTags)
  .setMeta(
    TRANSFORMER_METRIC.MEASUREMENT_TYPE.TRANSFORMATION.META.BAD_PARAM
  )
  .build();

```